docs-website-src/content/blockers.md

```diff
@@ -106,15 +106,15 @@ We block if your machine has multiple network interface cards (NICs) using kerne
   * The "default" approach in `network-scripts` config files of specifying NICs by `DEVICE` can cause issues due to the above.
   * To find a more in-depth explanation of *why* this is a problem (and what to do about it), read Freedesktop.org's <a href="https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/" target="_blank">Predictable Network Interface Names</a> documentation.
 
-One way to prevent these issues is to assign a custom name in the configuration and re-initialize NICs ahead of time.
+On systems configured in this manner, we will offer to automatically assign a custom names (from ethX to cpethX) in the configuration and re-initialize NICs ahead of elevation.
```

Comment:
Saying that we block on it, then saying that we offer to automatically update the NIC name is going to be confusing to anyone reading the documention.  I think we should update this section to something similar to the following:

```
### Multiple network interface cards using kernel names

If your machine has multiple network interface cards (NICs) using kernel names (`ethX`), we offer to automatically update the name from `ethX` to `cpethX` during elevation.
  * To find a more in-depth explanation of *why* this name change is necessary, read Freedesktop.org's <a href="https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/" target="_blank">Predictable Network Interface Names</a> documentation.
```